synapse/storage/data_stores/main/event_push_actions.py

```diff
@@ -37,6 +39,18 @@
 ]
 
 
+@attr.s
+class EventPushSummary(object):
+    """Summary of pending event push actions for a given user in a given room."""
+
+    user_id = attr.ib()
```

Comment:
right, for bonus points, you can give each of these a type.

```suggestion
    user_id = attr.ib(type=str)
```
```